### PR TITLE
Tailwind installed and configured with webpacker

### PR DESCRIPTION
**Pull request by Joberg95**
#### Description
Added tailwind css using webpacker.

#### What I have learned working on this feature:
How to install tailwind in an rails app.

### CODE REVIEW

File: app/javascript/css/tailwind.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/**
+ * This injects Tailwind's base styles, which is a combination of
+ * Normalize.css and some additional base styles.
+ *
+ * You can see the styles here:
+ * https://github.com/tailwindcss/tailwindcss/blob/master/css/preflight.css
+ *
```
Comment (amandagibson) [inline app/javascript/css/tailwind.css]: so many comments

Comment (nevroje) [reply app/javascript/css/tailwind.css]: yes and they are going to stay there for now

**Review by amandagibson — COMMENTED** [review]
remove unnecessary comments

File: .gitignore [hunk 2]
```diff
@@ -8,4 +8,8 @@
 /yarn-error.log
 
 .byebug_history
-/coverage
\ No newline at end of file
+/coverage/public/packs
+/public/packs-test
```
Comment (magnus-thor) [inline .gitignore:12]: add `/public/packs` in gitignore, and you have to remove it from git, `$ git rm -rf /public/packs`

File: coverage/.last_run.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (magnus-thor) [inline coverage/.last_run.json]: Remove the coverage folder from git and added it to .gitignore

**Review by magnus-thor — CHANGES_REQUESTED** [review]

**Review by magnus-thor — APPROVED** [review]
good job

**Pull request merged**